### PR TITLE
update: Softmax op

### LINKED ISSUE #3: [softmax] 算子调研
- framework
  - pytorch
  - paddle
  - onnx
- hardware
  - nvidia
    - cublas
    - cudnn
  - cambricon
  - kunlunxin

### PR DESCRIPTION
**Pull request by bitzyz**
- framework
  - pytorch
  - paddle
  - onnx
- hardware
  - nvidia
    - cudnn
  - cambricon
  - kunlunxin

### CODE REVIEW

File: operators/softmax.framework-onnx.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# Softmax in ONNX
+
+> - <https://onnx.ai/onnx/operators/onnx__Softmax.html>
```
Comment (YdrMaster) [inline operators/softmax.framework-onnx.md:3]: 一行可以不是无序列表

File: operators/softmax.framework-paddle.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Softmax in PaddlePaddle
+> <https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Softmax_cn.html>
```
Comment (YdrMaster) [inline operators/softmax.framework-paddle.md:2]: - 标题下面必须空一行；
- 多个连续的必须是无序表；

File: operators/softmax.framework-paddle.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+> <https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Softmax2D_cn.html>
+>
+> paddle框架中对齐pytorch，存在两种Softmax，一种是普适的Softmax，另一种是特定的Softmax2D
```
Comment (YdrMaster) [inline operators/softmax.framework-paddle.md:5]: 这个是你写的描述吧，不应该在引用块里

File: operators/softmax.framework-paddle.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+>
+## Softmax
+> **调用方法**
```
Comment (YdrMaster) [inline operators/softmax.framework-paddle.md:8]: - 空一行，同上；
- 不在引用块里，同上，而且这个好像应该是 ### 三级标题吧

File: operators/softmax.framework-pytorch.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Softmax in Pytorch
+
+> <https://pytorch.org/docs/stable/generated/torch.nn.Softmax.html>
```
Comment (YdrMaster) [inline operators/softmax.framework-pytorch.md:3]: 无序表，同上

File: operators/softmax.framework-pytorch.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+> <https://pytorch.org/docs/stable/generated/torch.nn.Softmax2d.html>
+>
+> pytorch中存在两种Softmax定义，一种是普适的Softmax，另一种是特定的Softmax2D
+## Softmax
+> **调用方法**
```
Comment (YdrMaster) [inline operators/softmax.framework-pytorch.md:8]: - 空一行，同上；
- 哪些是你写的，哪些是引用的？

File: operators/softmax.hardware-cambricon-1.21.1.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
+# Softmax in Cambricon
+
+> <https://www.cambricon.com/docs/sdk_1.15.0/cambricon_cnnl_1.21.1/developer_guide/cnnl_api/api/softmax.html#cnnlsoftmaxforward>
```
Comment (YdrMaster) [inline operators/softmax.hardware-cambricon-1.21.1.md:3]: 无序表，同上；

File: operators/softmax.hardware-cambricon-1.21.1.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+> <https://www.cambricon.com/docs/sdk_1.15.0/cambricon_cnnl_1.21.1/developer_guide/cnnl_api/api/softmax.html#cnnlsoftmaxforward-v2>
+
+## Softmax-v1
+**API**
```
Comment (YdrMaster) [inline operators/softmax.hardware-cambricon-1.21.1.md:7]: - 空一行，同上；
- 好像应该是标题，同上；

File: operators/softmax.hardware-kunlunxin.md [hunk 9]
```diff
@@ -0,0 +1,17 @@
+# Softmax in Kunlun
+
```
Comment (YdrMaster) [inline operators/softmax.hardware-kunlunxin.md:2]: 没写网址

File: operators/softmax.hardware-kunlunxin.md [hunk 10]
```diff
@@ -0,0 +1,17 @@
...
+## Softmax
+```c++
```
Comment (YdrMaster) [inline operators/softmax.hardware-kunlunxin.md:4]: 空一行，同上

File: operators/softmax.hardware-nvidia-cudnn-9.0.0.md [hunk 11]
```diff
@@ -0,0 +1,62 @@
+# Softmax in cudnn
+> <https://docs.nvidia.com/deeplearning/cudnn/api/cudnn-ops-library.html#cudnnsoftmaxforward>
```
Comment (YdrMaster) [inline operators/softmax.hardware-nvidia-cudnn-9.0.0.md:2]: 空一行，同上

File: operators/softmax.hardware-nvidia-cudnn-9.0.0.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
...
+
+## Softmax
+> **API**
```
Comment (YdrMaster) [inline operators/softmax.hardware-nvidia-cudnn-9.0.0.md:5]: - 空一行，同上；
- 哪些是引用的？

**Review by YdrMaster — CHANGES_REQUESTED** [review]

**Comment by YdrMaster** [comment]
建议用 vscode + markdown lint
